experiments/ClimaEarth/components/atmosphere/climaatmos.jl

```diff
@@ -378,8 +390,17 @@ The fields added are:
 - `:ustar`, `:L_MO`, `:buoyancy_flux` (for EDMF boundary conditions)
 """
 function Interfacer.add_coupler_fields!(coupler_field_names, atmos_sim::ClimaAtmosSimulation)
-    atmos_coupler_fields =
-        [:surface_direct_albedo, :surface_diffuse_albedo, :T_sfc, :q_sfc, :ρ_sfc, :ustar, :L_MO, :buoyancy_flux]
+    atmos_coupler_fields = [
+        :surface_direct_albedo,
+        :surface_diffuse_albedo,
+        :T_sfc,
+        :q_sfc,
+        :ρ_sfc,
+        :ustar,
+        :L_MO,
+        :buoyancy_flux,
+        :F_radiative,
```

Comment:
can you add a comment about why the atmosphere needs this in the docstring please?

Reply:
The function that needs it is
```
function update_sim!(sim::Interfacer.SurfaceModelSimulation, csf, area_fraction)
```
Should this be needed only by the surface models?